ccd-definition/CaseEvent/Gatekeeping.json

```diff
@@ -209,5 +209,33 @@
     "ShowSummary": "N",
     "ShowEventNotes": "N",
     "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "limitedUploadDocumentsGatekeeping",
+    "Name": "Documents",
+    "Description": "Upload documents",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "Gatekeeping",
+    "PostConditionState": "Gatekeeping",
+    "SecurityClassification": "Public",
+    "ShowSummary": "N",
+    "ShowEventNotes": "N",
+    "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "addStatementOfServiceGatekeeping",
+    "Name": "Add statement of service (c9)",
+    "Description": "Add statement of service",
+    "DisplayOrder": 13,
```

Comment:
I think display order with those two new elements is a bit off.

Reply:
done